packages/tkr-source-controller/bundle/config/values.yaml

```diff
@@ -2,6 +2,7 @@
 #@overlay/match-child-defaults missing_ok=True
 ---
 namespace: tkg-system
+legacyNamespace: tkr-system
```

Comment:
I think this line should also be added in tkg meta package's `values.xml`

Reply:
Good catch! Thanks @prkalle

Reply:
Done.